src/ZoneServer/Network/PacketHandler.cs

```diff
@@ -1632,12 +1633,20 @@ public void CZ_MAP_REVEAL_INFO(IZoneConnection conn, Packet packet)
 				return;
 			}
 
-			var revealedMap = new RevealedMap(mapId, visible, 0);
+			// Check the percentage for validity
+			if (percentage < 0 || percentage > 100)
+			{
+				Log.Warning("CZ_MAP_SEARCH_INFO: User '{0}' tried to update the visibility for map '{1}' beyond an acceptable percentage.", conn.Account.Name, mapId);
```

Comment:
This should've been updated to say `CZ_MAP_REVEAL_INFO`.

Reply:
My bad, I just copied it from the other method.